presto-tests/src/main/java/com/facebook/presto/tests/TestingPrestoClient.java

```diff
@@ -230,6 +230,9 @@ else if (type instanceof ArrayType) {
         else if (type instanceof DecimalType) {
             return new BigDecimal((String) value);
         }
+        else if (type.toString().equals("ObjectId")) {
```

Comment:
This should really do `type.getTypeSignature().getBase()` rather than relying on `toString()`. I also dislike putting a MongoDB-specific type in the `presto-tests` module, but I don't see a better way.